shared/src/msi-installer/ContinuousProfiler/EnvironmentVariables.wxs

```diff
@@ -5,10 +5,6 @@
   <?include $(sys.CURRENTDIR)\Config.wxi?>
   <Fragment>
     <ComponentGroup Id="ContinuousProfiler.EnvironmentVariables.Machine" Directory="INSTALLFOLDER">
```

Comment:
Does this do anything anymore? I'm not too familiar with WiX, but it looks like this does nothing now?

Reply:
> Does this do anything anymore?

Nope 😄 I left it in because it's referenced in `Project.wxs` so it would make it easier to add variables later if we _do_ need to add them

Reply:
Sounds good!